src/components/Button.spec.js

```diff
@@ -0,0 +1,9 @@
+import { render, screen } from "@testing-library/react";
+import Button from "./Button";
+
+describe("Button", () => {
+  it("has text and shows the way back", () => {
```

Comment:
You don't test whether the button is showing the way back. You only test, if the button renders with text.
```suggestion
  it("renders a button with text", () => {
```